### ISSUE TITLE
[Bug] Area Chart - Tooltip Conflicts with Emphasis

### ISSUE THREAD

**Issue by hicksoft** [apache/echarts#20630]
### Version

5.6.0

### Link to Minimal Reproduction

https://echarts.apache.org/examples/en/editor.html?c=area-stack

### Steps to Reproduce

1. Using the current Stacked Area Chart example

### Current Behavior

When hovering over a series, emphasis applies to fade the other series and the tooltip appears. 
Once this happens, move the cursor within the same series. When you hit another data point for the series, the tooltip remains but emphasis stops applying.

### Expected Behavior

Emphasis should continue being applied as long as the user's cursor is over the series

### Environment

```markdown
n/a
```

**Comment by helgasoft** [comment]
> When you hit another data point for the series, the tooltip remains but emphasis stops applying.

I do not think it is a bug, because **focus** has not been defined rigorously. And _focus_ behavior makes sense:
When ```tooltip: { trigger:'item'}``` focus remains for the entire area of one series.
When ```tooltip: { trigger:'axis'}``` focus is dependent on tooltip which has to emphasize single points for all series, and that cannot happen if series are blurred.

**Comment by hicksoft** [comment]
For `{ trigger:'axis'}`, I don't think that behaviour does make sense.

When you first enter the chart, both the focus and tooltip apply. When you switch from one series to another, both the focus and tooltip apply. It's unexpected that the behaviour would be different when moving left-right (ie within a series) vs moving up-down (ie between series)

**Comment by helgasoft** [comment]
What should happen to the tooltip when moving left-right ? Default tooltip behavior is to **emphasize all** series points. But **only one** series in in focus, the others are blurred. So should all affected points be shown while keeping their series blurred? Or not show them at all? What behavior would be expected and acceptable for both focus and tooltip?

**Comment by hicksoft** [comment]
I think that only the one series should be in focus but all series should show on the tooltip.

The reason I say this, is just because that's the initial behaviour that occurs when brining your cursor onto the chart right now. Anything else produces an inconsistent experience.

If the tooltip were to only show the highlighted series, then the initial tooltip would be the bug as it shows all of the series while only highlighting the one.

Either of these approaches are logical but right now the behaviour toggles back and forth which is not expected from a user experience standpoint. For my personal use case, I'd prefer highlighting the one series and showing all on the tooltip
